src/databricks/sqlalchemy/test_local/e2e/test_basic.py

```diff
@@ -463,3 +499,45 @@ def test_get_columns(db_engine, sample_table: str):
     columns = inspector.get_columns(sample_table)
 
     assert True
+
+
+class TestCommentReflection:
+    @pytest.fixture(scope="class")
+    def engine(self):
+        HOST = os.environ.get("host")
+        HTTP_PATH = os.environ.get("http_path")
+        ACCESS_TOKEN = os.environ.get("access_token")
+        CATALOG = os.environ.get("catalog")
+        SCHEMA = os.environ.get("schema")
+
+        connection_string = f"databricks://token:{ACCESS_TOKEN}@{HOST}?http_path={HTTP_PATH}&catalog={CATALOG}&schema={SCHEMA}"
+        connect_args = {"_user_agent_entry": USER_AGENT_TOKEN}
+
+        engine = create_engine(connection_string, connect_args=connect_args)
+        return engine
+
+    @pytest.fixture
+    def inspector(self, engine: Engine) -> Inspector:
+        return Inspector.from_engine(engine)
+
+    @pytest.fixture
+    def table(self, engine):
+        md = MetaData()
+        tbl = Table(
+            "foo",
+            md,
+            Column("bar", String, comment="column comment"),
+            comment="table comment",
+        )
+        md.create_all(bind=engine)
+
+        yield tbl
+
+        md.drop_all(bind=engine)
+
+    def test_table_comment_reflection(self, inspector: Inspector, table: Table):
+        tbl_name = table.name
+
+        comment = inspector.get_table_comment(tbl_name)
+
+        assert comment == {"text": "table comment"}
```

Comment:
Could also check column comments with `inspector.get_columns` ?

Reply:
Yes, you already did this with the `test_column_comments`, no? I can rewrite it to use the same fixtures and be more efficient.

Reply:
woof, haven't had my coffee yet. i understand your comment now. I've added an extra method to this test class that simply checks for reflection of the column comment. thank you!

Reply:
Awesome 👍 !